test/xpu_api/numerics/complex.number/cmplx.over/pow.pass.cpp

```diff
@@ -84,26 +84,18 @@ ONEDPL_TEST_NUM_MAIN
     test<unsigned, float>();
     test<long long, float>();
 
-    auto fnc1 = []()
-    {
-        test<int, double>();
-        test<unsigned, double>();
-        test<long long, double>();
-        test<float, double>();
-        test<double, float>();
-    };
-    IF_DOUBLE_SUPPORT(fnc1);
```

Comment:
This line expanded to
TestUtils::invoke_test_if(HasDoubleSupportInRuntime(), [](){ fnc1; });
but fnc1 was not captured.

Reply:
And the root cause of this approach was that in previous implementation of this defines we are not able to transfer into this macro somthing with comma inside.